ScrollCounter/NumberScrollCounter.swift

```diff
@@ -278,6 +284,20 @@ public class NumberScrollCounter: UIView {
         
         let startingX = startingXCoordinate
         let seperatorLocation = digitScrollers.count - decimalPlaces
+
```

Comment:
I am not satisfied with this solution at all. If you have better idea, please share it with me!